node-repository/src/main/java/com/yahoo/vespa/hosted/provision/autoscale/ClusterModel.java

```diff
@@ -110,8 +108,8 @@ public double queryFractionOfMax() {
         return queryFractionOfMax = clusterTimeseries().queryFractionOfMax(scalingDuration(), clock);
     }
 
-    /** Returns average load during the last {@link #currentLoadDuration} */
-    public Load currentLoad() { return nodeTimeseries().averageLoad(clock.instant().minus(currentLoadDuration)); }
```

Comment:
Is the `averageLoad` method still used or can it be removed?

Reply:
Still used.